src/Compilers/CSharp/Portable/Binder/Binder_Statements.cs

```diff
@@ -451,7 +451,7 @@ private BoundStatement BindLocalFunctionStatement(LocalFunctionStatementSyntax n
 
                 if (block != null)
                 {
-                    localSymbol.ComputeReturnType(block, true);
+                    localSymbol.ComputeReturnType(block, true, false);
```

Comment:
Can we use named arguments here?  Reading `true, false` doesn't convey a lot of information for those unfamiliar with the API (namely, ... me :smile:).